tools/package.sh

```diff
@@ -6,7 +6,7 @@ rm -f ./*.vsix
 yarn run webpack
 VERSION="$(./tools/get-marketplace-version.sh)"
 vsce_package_args=(--no-dependencies --no-git-tag-version --no-update-package-json --readme-path docs/README.md)
-if [[ "$VERSION" != *.0 ]]; then
+if [[ "$VERSION" != *.0 && "$VERSION" != *00000000 ]]; then
```

Comment:
Could we replace this long random digit with just the date? e.g: `date -u +"%Y%m%dT%H%M%S"`

Reply:
As long as a new version number is greater than any of existing version number, it will work.  So just using numbers based on the current date/time should work, but my intention here is to 

(1) make distinction between pre-releases and official releases, and 
(2) provide an easy way to co-relate official releases and GitHub releases

i.e. we can easily identify if a version is official by looking last digits of the patch version and checking release notes on the GitHub page is also easy, i.e. look at the release note for GitHub version `24.6.1` for Marketplace version `24.6.100000000`.